packages/api/src/functions/dbAuthErrors.js

```diff
@@ -0,0 +1,88 @@
+export const WrongVerbError = class extends Error {
```

Comment:
Rob, what do you think of directly exporting the class instead of assigning them to constants? e.g. -> 

```js
export class WrongVerbError extends Error {
  constructor(properVerb) {
    super(`Only accessible via ${properVerb}`)
    this.name = 'WrongVerbError'
  }
}
```

Reply:
So many different ways to export things! Yeah that looks nicer.